tests/Doctrine/Migrations/Tests/InlineParameterFormatterTest.php

```diff
@@ -54,11 +58,13 @@ public function testFormatParameters(): void
             'unknown',
             'unknown',
             'unknown',
+            ParameterType::STRING,
+            ArrayParameterType::INTEGER,
```

Comment:
```suggestion
            class_exists(ArrayParameterType::class) ? ArrayParameterType::INTEGER : Connection::PARAM_INT_ARRAY,
```

This way, we can avoid to bump dependencies in a bugfix release.

Reply:
pushed, and this is what I was worried about - phpstan failure... :O do you want to me to add ignore on line?

Reply:
Yes, we can ignore the error.

Reply:
@derrabus pushed, green now.

any chance to get it released soon? thanks